api-report/container-loader.api.md

```diff
@@ -93,12 +93,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     // (undocumented)
     proposeCodeDetails(codeDetails: IFluidCodeDetails): Promise<boolean>;
     raiseContainerWarning(warning: ContainerWarning): void;
-    // @deprecated
-    get readonly(): boolean | undefined;
     // (undocumented)
     get readOnlyInfo(): ReadOnlyInfo;
```

Comment:
Please add a corresponding documentation block to readOnlyInfo even if it's pretty simple.

Reply:
Maybe something like:

```suggestion
    /**
      * Object defining the current readonly state of the Container.
      * If the Container is readonly it also provides additional properties describing how it got to that state.
     **/
    get readOnlyInfo(): ReadOnlyInfo;
```